app/controllers/items_controller.rb

```diff
@@ -1,4 +1,26 @@
 class ItemsController < ApplicationController
+  before_action :authenticate_user!, only: [:new]
+  
   def index
+
   end
+
+  def new
+    @item  = Item.new
+  end
+
+  def create
+    @item = Item.create(item_params)
```

Comment:
```suggestion
    @item = Item.new(item_params)
```
上記のように変更をお願い致します。
createメソッドはnew+saveの役割をしているためです。